evalml/tests/component_tests/test_polynomial_detrender.py

```diff
@@ -0,0 +1,94 @@
+import numpy as np
+import pandas as pd
+import pytest
+import woodwork as ww
+from sklearn.linear_model import LinearRegression
+from sklearn.preprocessing import PolynomialFeatures
+
+from evalml.pipelines.components import PolynomialDetrender
+
+pytest.importorskip('sktime', reason='Skipping polynomial detrending tests because sktime not installed')
+
+
+def test_polynomial_detrender_init():
+    delayed_features = PolynomialDetrender(degree=3)
+    assert delayed_features.parameters == {"degree": 3}
+
+
+def test_polynomial_detrender_raises_value_error_target_is_none(ts_data):
+    X, y = ts_data
+
+    with pytest.raises(ValueError, match="y cannot be None for PolynomialDetrender!"):
```

Comment:
Realllly nit-picking now but do we want to add tests for fit/transform too for completeness :')